src/emulator/gui/src/ui_main.cpp

```diff
@@ -119,5 +119,8 @@ void DrawUI(HostState &host) {
     if (host.gui.allocations_dialog) {
         DrawAllocationsDialog(host);
     }
+	if (host.gui.about_dialog) {
+        DrawAboutDialog(host);
+	}
```

Comment:
Don't mind me if I'm being a little too picky but... isn't this indentation a little weird?

Reply:
being picky is fine, and i don't really know what happened there lol